game.js

```diff
@@ -0,0 +1,212 @@
+  cardDom.querySelector('.card-value').innerText = card.value
+  cardDom.querySelector('.suit-top').innerHTML = `<img src="images/${card.suit}.png"  alt="${card.suit}">`
+  cardDom.querySelector('.suit-bottom').innerHTML = `<img src="images/${card.suit}.png" alt="${card.suit}">`
+  cardDom.style.display = 'flex';
+}
+
+function DrawNewDeckcards() {
+  for (dealerCardIndex = 0; dealerCardIndex < 2; dealerCardIndex++) {
+    let suit = 1 + Math.trunc(Math.random() * deck.length)
+    let card = deck[suit]
+    dealerCards.push(getcardstring(card))
+
+    dealerTotal += defineNum(card);
+    ShowPoker(card, dealerCardIndex, '.dcardspic')
+  }
+
+  for (playerCardIndex = 0; playerCardIndex < 2; playerCardIndex++) {
+    let suit = 1 + Math.trunc(Math.random() * deck.length)
+    let card = deck[suit]
+
+    playerCards.push(getcardstring(card))
+    playerTotal += defineNum(card);
+    ShowPoker(card, playerCardIndex, '.pcardspic')
+  }
+}
+
+
+
+function Newgamedeck() {
+  DrawNewDeckcards()
+
+  Playercardsinfo.innerHTML = `<p id="Playercardsinfo">Player Cards: ${playerCards} -- Total: ${playerTotal}<p>`
+
+  checkgamestatus();
+
+}
+
+function buttonChange() {
+  hitbutton.style.display = "none"
+  staybutton.style.display = "none";
+  gameoverbutton.style.display = "inline"
+
+}
+
+function checkgamestatus() {
+  if (playerTotal == 21) {
+    gameover = true
+    console.log("You Win!")
+    Gamestatus.innerText = "You Win!";
+    buttonChange()
+
+  } else if (dealerTotal == 21) {
+    gameover = true
+    console.log("Dealer Win!")
+    Gamestatus.innerText = "Dealer Win!";
+    buttonChange()
+
+  } else if (dealerTotal > 21) {
+    gameover = true
+    console.log("You Win!")
+    Gamestatus.innerText = "You Win!";
+    buttonChange()
+  } else if (playerTotal > 21) {
+    gameover = true
+    console.log("Dealer Win!")
+    Gamestatus.innerText = "Dealer Win!";
+    buttonChange()
+  } else {
+    gameover = false
+    console.log("Game Continues");
+  }
+
+}
+
+function callEndgame() {
+  if (playerTotal > dealerTotal <= 21) {
+    gameover = true
+    Gamestatus.innerText = "Dealer Win!";
+    console.log('playercardstotal>dealercardstotal<=21')
+    buttonChange()
+    //calculation breaks when dealer chose hit first and then stay, no matter the result it will show Dealer win
+  } else if (dealerTotal == playerTotal) {
+    gameover = true
+    Gamestatus.innerText = "Tie";
+    buttonChange()
+  } else if (dealerTotal > playerTotal <= 21) {
+    gameover = true
+    Gamestatus.innerText = "You Win!";
+    buttonChange()
+  }
+  else {
+    gameover = true
+    Gamestatus.innerText = "You Win!";
+    buttonChange()
+  }
+
+}
+function DealerGetNewCards() {
+  if (dealerTotal <= 15) {
+    let Dcard = 1 + Math.trunc(Math.random() * deck.length);
+    let tmp = deck[Dcard]
+    dealerCards.push(getcardstring(tmp));
+    dealerTotal += defineNum(tmp);
+    ShowPoker(tmp, dealerCardIndex, '.dcardspic');
```

Comment:
add dealercardIndex,classname in ShowPoker(), Give dcardspic[dealerCardIndec] a style. 
add dealerCardIndex++
remove return